### ISSUE TITLE
In dark theme Keyboard focus is not visible on "Clear task" button: A11y_.NET maui_User can get all the insights of Dashboard_Kayboard

### ISSUE THREAD

**Issue by kapilvaishna** [dotnet/maui#30805]
**Please do not close this bug. This bug should only be closed by Trusted Tester after verification.**
"[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier."

## GitHub Tags
#A11yTCS; #A11yMAS; #WCAG2.4.7; #Keyboard; #BM_.NETMaui_Win32_JUL2025; #DesktopApp; #Win32; #FTP; #A11ySev2; #.NET MAUI SDKs; 

## Environment Details:
Application: Developer Balance (.NET Maui)
Windows 11
OS Build: 26100.2161
Version 24H2

## Repro Steps:

1. Install and open the Developer Balance app
2. Apply dark theme
3. TAB till "Clear task" button
4. Observe the issue 
## Actual Result:
After applying dark theme when we TAB to "Clear task" button. keyboard focus is not visible.
Note: Issue seen through out the application for "Clear task" button.

## Expected Result:
After applying dark theme  when we TAB to "Clear task" button Keyboard focus should visible.

## User Impact:
Low vision user who uses application in dark theme will unable to see the keyboard focus.

## Attachment

https://github.com/user-attachments/assets/4beb3d21-642e-420b-a16b-92a7d9c4b108

**Comment by ghost** [comment]
This issue has been verified in Visual Studio 17.14.0 Preview 4.0 with MAUI versions (9.0.0 & 9.0.90). It can be reproduced on Windows platform.

**Comment by jsuarezruiz** [comment]
@sheiksyedm  I noticed this has been reopened, could you share the current status? Let me know if there’s any way I can help.

**Comment by sheiksyedm** [comment]
@jsuarezruiz  Shane informed that accessibility issues can only be closed by the Accessibility team after verifying the fix. We can add the a11y-resolved label to indicate the issue has been addressed.

**Comment by AnKushSingh05** [comment]
#Closed;#A11yFixed;
Issue is fixed

<img width="1366" height="768" alt="Image" src="https://github.com/user-attachments/assets/0fc680cc-f6da-4766-9fdc-7f5d30707006" />
